accessibility/README.md

```diff
@@ -126,6 +126,13 @@ assign it one of four ratings:
 Use the Notes sheet to leave per-cell comments when necessary, referencing them
 with a link. The next steps for an audit are handled on a per-project basis.
 
+## Translations
+
+- Ensure that the application is setup to support multiple locales.
+- Ensure that the application raises an error when a translation is missing for a
+  given locale in development.
```

Comment:
Should it raise on tests?

Reply:
I think [it does](https://github.com/thoughtbot/suspenders/blob/55cb5c246fe53aea4cf53436c7c208b40ad13e85/NEWS.md?plain=1#L465). We'll have to adjust the wording here.